src/styles/index.scss

```diff
@@ -1,3 +1,62 @@
+@import './utils/variable';
+
 body {
   margin: 0;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  font-size: $main-font-size;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.calendar {
+  display: flex;
+  align-items: center;
+  flex-wrap: wrap;
+  gap: $main-gap;
+
+  width: $main-width;
+  padding: $padding-calendar;
+
+  @each $day, $value in $days-of-week {
+    &--start-day-#{$day} > :first-child {
+      margin-left: ($width-days * $value) + $value;
```

Comment:
The calculation of the `margin-left` seems incorrect. According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), calculations should be wrapped in parentheses for clarity. Also, ensure that the `$value` variable is being used correctly in the calculation, as it might not be providing the correct margin without proper multiplication or addition with units.